src/tsickle.ts

```diff
@@ -106,6 +106,16 @@ function hasModifierFlag(node: ts.Node, flag: ts.ModifierFlags): boolean {
   return (ts.getCombinedModifierFlags(node) & flag) !== 0;
 }
 
```

Comment:
This patch's description describes what it does but not why.  It's not clear to me why it's needed.